src/win/tty.c

```diff
@@ -2255,25 +2250,67 @@ int uv_tty_reset_mode(void) {
   return 0;
 }
 
+static int uv__set_vtp(HANDLE handle, BOOL vtp) {
+  DWORD dwMode = 0;
+
+  if (!GetConsoleMode(handle, &dwMode)) {
```

Comment:
The preferred style is to drop the curly braces on `if` statements whose bodies are a single statement. Can you apply those changes throughout the PR.